### ISSUE TITLE
Deploy osx build from non osx systems (Symlinks broken?)

### ISSUE THREAD

**Issue by s-a** [electron-userland/electron-packager#97]
So far attempts failed. Just tried to copy osx build folder on ubuntu. Same behavior as on Windows

```
Warning: Unable to read "dist/osx64/sulu.app/Contents/Frameworks/Electron Framework.framework/Frameworks"
```

Anyway it is possible to zip the output folder on ubuntu, but as far as I know a symlink is a symlink so i am quite unsure if these symlinks are not realy broken. Where do these symlinks point to? Isnt it any folder within the project build or are these folder some points on the osx target machine?

![image](https://cloud.githubusercontent.com/assets/1270662/8516049/e74ac18a-23ae-11e5-81a4-e7eeecb6515b.png)

**Comment by dorian-marchal** [comment]
Same issue here :+1:

**Comment by KrisSiegel** [comment]
Same problem; attempting to package up an application for Darwin fails due to

```bash
Cannot create symlinks; skipping darwin platform
```

**Comment by neal-morgansolar** [comment]
Also having this issue - is there still no way to build osx apps on non-osx systems?

**Comment by maxogden** [comment]
If someone is able to write a failing test for this it would be super helpful

**Comment by dogancelik** [comment]
#### This works

````sh
electron-packager . MyApp --platform=all --arch=x64 --version=0.36.7 --out=build --ignore=\"(build|node_modules)\" --overwrite
```

#### This does NOT work

```sh
electron-packager . %npm_package_productName% --platform=all --arch=x64 --version=%npm_package_electronVersion% --out=build --ignore=\"(build|node_modules)\" --overwrite
```

The only difference is NOT working command uses variables: `%npm_package_productName%` and `%npm_package_electronVersion%`

In *package.json*:

```json
{
  "productName": "MyApp",
  "electronVersion": "0.36.7"
}
```

Windows 8, npm 3.3.2, node 5.3.0, electron-packager 5.2.1

**Comment by malept** [comment]
Not being able to create symlinks on Windows as a non-admin is a known limitation - code was added early on to check whether a given host can create symlinks and responds with the "Cannot create symlinks" error message if it can't. I've committed https://github.com/electron-userland/electron-packager/commit/fb8c181b3d772ba1fc6b761f3da542c317561c5d to add a reason why it cannot create symlinks in the error message.

As for building an macOS target on a Linux host, I cannot currently reproduce this behavior. It may have been an issue with the Electron version used.
